src/Lykke.blue.Service.ReferralLinks/Startup.cs

```diff
@@ -91,6 +96,20 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplica
                     app.UseDeveloperExceptionPage();
                 }
 
+                app.UseLykkeMiddleware(Constants.ComponentName, ex =>
+                {
+                    Log.WriteFatalErrorAsync(Constants.ComponentName, "", ex);
```

Comment:
why fatal error?
why ""?

Reply:
Ok, I changed it to:
Log.WriteErrorAsync(Constants.ComponentName, "Unexpected error", ex);
return ErrorResponseModel.Create("Technical problem");

The reason for the empty string was that we don't know the exact context where the error occured, I set it to "Unexpected error" which is not that helpful either. Probably more info on the context where the exception occurred can be retrieved from the stack trace which gets logged from the exception anyway.